agent/config/config.go

```diff
@@ -80,6 +80,9 @@ const (
 	// minimumNumImagesToDeletePerCycle specifies the minimum number of images that to be deleted when
 	// performing image cleanup.
 	minimumNumImagesToDeletePerCycle = 1
+
+	// Default cgroup prefix for ECS tasks
+	DefaultTaskCgroupPrefix = "/ecs"
```

Comment:
Will this be configurable by environment variable or task definition, I didn't see that in the code?